response_operations_ui/controllers/notify_controller.py

```diff
@@ -39,15 +37,17 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
             notification.update({"reference": reference})
 
         url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
+        status_number = response.status_code
+
+        if status_number == 201:
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json().get("id"))
         else:
             ref = reference if reference else 'reference_unknown'
-            raise NotifyError("There was a problem sending a notification to Notify-Gateway to GOV.UK Notify",
-                              reference=ref)
+            logger.error(f'STATUS CODE: {status_number}')
+            logger.error(f'URL: {url}')
+            raise NotifyError(f'FAILED TO SEND EMAIL TO: {notification["emailAddress"]}', reference=ref)
```

Comment:
The message in this isn't great and it lacks the context you'd need for a useful exception to be thrown (which is why we're fixing this in the first place).  Also never just put an error message in caps, it's not helpful.
The logging that was there before was fine, but `"There was a problem sending a notification to the notify-gateway"` is probably a little better because it's not needlessly wordy.  If you look at the NotifyError in exceptions.py, you'll notice it takes an error, a description and any other number of kwargs (key value pairs essentially, e.g., raise NotifyError("Text", response.text, key1=value1, key2=value2, etc)).

This should probably be something like 
```
raise NotifyError("There was a problem sending a notification to the notify-gateway", response.text, reference=ref)
```
I've not tested any of my suggestions btw so there's a chance that it might not be as easy as what I've mentioned, but it'll be a useful starting point